JS1064_JS104_create_do_while_loop/script.js

```diff
@@ -0,0 +1,8 @@
+// Display the prompt dialogue while the value assigned to `secret` is not equal to "sesame"
+let password = 'abc123';
+do {
+  prompt("What is the secret password?");
```

Comment:
should assign the prompt value to the `password` variable, to update and match the condition
```suggestion
  password = prompt("What is the secret password?");
```